pkgs/by-name/sc/scmutils/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+  mitschemeX11,
+  writeShellScriptBin,
+}:
+stdenv.mkDerivation rec {
+  pname = "scmutils";
+  version = "20230902";
+
+  src = fetchurl {
+    url = "https://groups.csail.mit.edu/mac/users/gjs/6946/mechanics-system-installation/native-code/${pname}-src-${version}.tar.gz";
+    hash = "sha256-9/shOxoKwJ4uDTHmvXqhemgy3W+GUCmoqFm5e1t3W0M=";
+  };
+
+  buildInputs = [ mitschemeX11 ];
+
+  configurePhase = ''
+    ln -r -s kernel/ghelper-pro.scm kernel/ghelper.scm
+    ln -r -s solve/nnsolve.scm solve/solve.scm
+    substituteInPlace load.scm \
+      --replace-fail '/usr/local/scmutils/' "$out/lib/mit-scheme/"
+  '';
+
+  buildPhase = ''
+    echo '(load "compile")' | mit-scheme --no-init-file --batch-mode --interactive
+    echo '(load "load") (disk-save "edwin-mechanics.com")' | mit-scheme --no-init-file --batch-mode --interactive
```

Comment:
```suggestion
    runHook preBuild
    echo '(load "compile")' | mit-scheme --no-init-file --batch-mode --interactive
    echo '(load "load") (disk-save "edwin-mechanics.com")' | mit-scheme --no-init-file --batch-mode --interactive
    runHook postBuild
```

Reply:
I'll update the commit to include these changes.